src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,35 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      key={id}
+      className={cn('TodoInfo', {
+        'TodoInfo--completed': completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      {user && (
+
+        <UserInfo user={user} />
```

Comment:
```suggestion
      {user && (
        <UserInfo user={user} />
```

Reply:
<img width="646" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/b463367e-75ae-494a-a985-e07217b6c966">
still not fixed